core/trino-main/src/main/java/io/trino/sql/analyzer/Analysis.java

```diff
@@ -1874,6 +1874,7 @@ public static class MergeAnalysis
         private final List<List<ColumnHandle>> mergeCaseColumnHandles;
         // Case number map to columns
         private final Multimap<Integer, ColumnHandle> updateCaseColumnHandles;
+        private final Map<ColumnHandle, io.trino.sql.ir.Expression> defaultColumnValues;
```

Comment:
A question not relates to this : I see that the value is assigned to ir.Expression here. Could you clarify the boundary between when we should use ir.Expression vs. Expression? I’m trying to understand the intended separation or convention

Reply:
And does it now actually already to be the `Constant` ?